custom_cards/custom_card_haven_washer_dryer/custom_card_haven_washer_dryer.yaml

```diff
@@ -0,0 +1,580 @@
+
+            if (variables.ulm_custom_card_washer_dryer_job_state) {
+              rows.push("\'row2\'");
+            }
+
+            if (variables.ulm_custom_card_washer_dryer_remote_control) {
+              rows.push("\'row3\'");
+            }
+
+            if (variables.ulm_custom_card_washer_dryer_delayed_start
+              && variables.ulm_custom_card_washer_dryer_delayed_starttime) {
+              rows.push("\'row4\'");
+            }
+
+            return rows.join(" ");
+          ]]]
+      - grid-template-columns: "1fr"
+      - grid-template-rows: "min-content"
+      - row-gap: "0px"
+    custom_fields:
+      row2:
+        - display: >
+            [[[
+              return (variables.ulm_custom_card_washer_dryer_job_state
+                && variables.ulm_custom_card_washer_dryer_power
+                && states[variables.ulm_custom_card_washer_dryer_power] !== undefined
+                && states[variables.ulm_custom_card_washer_dryer_power].state === "on") ? "block" : "none";
+            ]]]
+      row3:
+        - display: >
+            [[[
+              return (variables.ulm_custom_card_washer_dryer_power
+                && states[variables.ulm_custom_card_washer_dryer_power] !== undefined
+                && states[variables.ulm_custom_card_washer_dryer_power].state === "on"
+                && variables.ulm_custom_card_washer_dryer_remote_control
+                && states[variables.ulm_custom_card_washer_dryer_remote_control] !== undefined
+                && String(states[variables.ulm_custom_card_washer_dryer_remote_control].state) === "true") ? "block" : "none";
+            ]]]
+      row4:
+        - display: >
+            [[[
+              return (variables.ulm_custom_card_washer_dryer_power
+                && states[variables.ulm_custom_card_washer_dryer_power] !== undefined
+                && states[variables.ulm_custom_card_washer_dryer_power].state === "on"
+                && variables.ulm_custom_card_washer_dryer_remote_control
+                && states[variables.ulm_custom_card_washer_dryer_remote_control] !== undefined
+                && String(states[variables.ulm_custom_card_washer_dryer_remote_control].state) === "true"
+                && variables.ulm_custom_card_washer_dryer_delayed_start
+                && states[variables.ulm_custom_card_washer_dryer_delayed_start] !== undefined
+                && states[variables.ulm_custom_card_washer_dryer_delayed_start].state === "on"
+                && variables.ulm_custom_card_washer_dryer_delayed_starttime) ? "block" : "none";
+            ]]]
+  custom_fields:
+    row1:
+      card:
+        type: "custom:button-card"
+        styles:
+          card:
+            - box-shadow: "none"
+            - padding: "0px"
+        template:
+          - "icon_info"
+          - "blue_on"
+        tap_action:
+          action: "more-info"
+        entity: "[[[ return variables.ulm_custom_card_washer_dryer_power; ]]]"
+        label: >
```

Comment:
Have you considered adding variables or using the translating engine to translate the strings of the states?

Reply:
Thank you for your time for reviewing my pull request. I tried to keep the card as simple as possible. The friendly name and icon of the power switch is used as the name and icon of the card. The label changes to the text you supply using the 3 variables available. Could you explain how I could use the translation engine is this case?

Reply:
Added translation on the default state of the washing machine